Import UI fixes

jayoshih opened the pull request:
Addresses the following:
* [Indeterminate status of "select all" not showing in import from channel flow](https://www.notion.so/learningequality/Indeterminate-status-of-select-all-not-showing-in-import-from-channel-flow-d3d28728e72a4cb0a13c5284d08564b9)
* [Card layout & typography is inconsistent](https://www.notion.so/learningequality/Card-layout-typography-is-inconsistent-e8a1289e003b4d90b367e1fcb985a15d)
* [Change description to "caption" color and reduce visual prominence of "Show more"](https://www.notion.so/learningequality/Change-description-to-caption-color-and-reduce-visual-prominence-of-Show-more-f1da77ec9a7a46bdb212d0674bf564da)
* [Import from other channels needs dir="auto" on cards](https://www.notion.so/learningequality/Import-from-other-channels-needs-dir-auto-on-cards-77ef05188125477ba63b63ab94ba3f50)
* [Way too big font size for topics in import modal ](https://www.notion.so/learningequality/Way-too-big-font-size-for-topics-in-import-modal-e34550979d884b2494cb1ba3111b6ec5)
* [Import from channel cards need interaction design rework. The inconsistency of what happens when you click topic cards vs resource cards in import from channels is really confusing](https://www.notion.so/learningequality/Import-from-channel-cards-need-interaction-design-rework-The-inconsistency-of-what-happens-when-you-6a00d45fe03b414685367b86af5ccf0c)
* [Small alignment issues in import from channel, search button a little small compared to text field?](https://www.notion.so/learningequality/Small-alignment-issues-in-import-from-channel-search-button-a-little-small-compared-to-text-field-64d657ee46ca406393a2847367d2d7ad)
* Fixes https://github.com/learningequality/studio/issues/1991
* [Resource drawer overlay is behind fullscreen modal](https://www.notion.so/learningequality/Resource-drawer-overlay-is-behind-fullscreen-modal-958ab3bd9ef94607ae75d7c47ada1294)

micahscopes commented on contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/BrowsingCard.vue:
```diff
@@ -230,4 +239,8 @@
     }
   }
 
+  .card-header {
```
This class doesn't seem to be applied, so the header `font-size` is ending up as 20px instead of 18px

jayoshih replied:
Nice catch! Removed it

micahscopes reviewed (changes requested):
For the most part, everything is looking good.  I did notice a few things though:

- Content card headings are still at `font-size` 20px (the class isn't being added anywhere)

- It could be nice for there to be a small margin between the search bar and the search button right now they're connected:
![image](https://user-images.githubusercontent.com/389782/96172742-769cf280-0eec-11eb-8e26-b59744d4ebe9.png)

- After opening a topic that is selected for import, I can no longer preview its content items by clicking on their cards:
![clicking selected content doesn't open previewer](https://user-images.githubusercontent.com/389782/96173860-127b2e00-0eee-11eb-9176-0c4ade716aaf.gif)

This one is kinda minor, and maybe not a big deal for now... clicking the "info" icon for a different topic doesn't open that topic's sidebar if another topic's sidebar is already open, so that you have to click twice:
![gotta click twice](https://user-images.githubusercontent.com/389782/96173964-3e96af00-0eee-11eb-9e30-103783d881b5.gif)

Other than that, everything seems to be working very well!

jayoshih commented:
> It could be nice for there to be a small margin between the search bar and the search button right now they're connected:

That is the intended design- we wanted to make it look like a search bar

> After opening a topic that is selected for import, I can no longer preview its content items by clicking on their cards:

Nice catch! Fixed

> This one is kinda minor, and maybe not a big deal for now... clicking the "info" icon for a different topic doesn't open that topic's sidebar if another topic's sidebar is already open, so that you have to click twice:

Ah that was probably related to [this issue](https://github.com/learningequality/studio/issues/1991). Fixed!

micahscopes reviewed (approved):
Feel free to merge once you hear back on what the correct font size is!
